app/models/views/application_overview.rb

```diff
@@ -94,6 +94,13 @@ def reference
       @application.reference if evidence_check_or_part_payment?
     end
 
+    def decision_type
+      {
+        'evidence_check' => 'evidence',
+        'part_payment' => 'payment'
```

Comment:
No, there isn't. But the class is an `ApplicationOverview` and the method is called `decision_type`, which doesn't suggest you can't use it for an `Application` object itself, if that makes sense.